site/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "start": "npm run sync && run-p start:*",
     "build": "npm run clean && run-s build:*",
-    "build:docs": "DOCS_GEN=TRUE node ../scripts/docs.js",
```

Comment:
What does this ENV var do again?

Reply:
Removes the special characters made by chalk. But we don't need to set it before calling docs.js because it's set within the script itself: https://github.com/netlify/cli/blob/master/scripts/docs.js#L7

Reply:
re slack discussion: we set this in the script file.  This way we can avoid cross-env https://github.com/netlify/cli/pull/140/files#diff-032384f2d383805f5dabad18c6e37353L7